examples/tokyo_travel_planner.py

```
@@ -0,0 +1,100 @@
+- Shopping areas (Harajuku, Ginza)
+- Day trips (Nikko, Kamakura, Mt Fuji)
+
+Make it practical and specific with actual place names, not generic suggestions."""
+
+    try:
+        # Make direct LLM call
+        request = LLMRequest(
+            provider=provider,
+            user_prompt=prompt,
+            temperature=0.7,
+            max_tokens=4000
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)